### PR TITLE
Skylar-mobile-check

### PR DESCRIPTION
**Pull request by sky1113**
checks if user is on mobile and throws an error

### CODE REVIEW

**Comment by JamesPHoughton** [comment]
I believe I'm waiting on a cypress test of this before we merge? Does it seem like a straightforward thing to do, or will it be a lot of work?

**Comment by sky1113** [comment]
The test has been added and updated to pass!

File: cypress/e2e/Mobile_Check.js
Comment (JamesPHoughton) [inline cypress/e2e/Mobile_Check.js:5]: the name of the test set wants to be changed from "normal_paths" to "mobile_check"

File: cypress/e2e/Mobile_Check.js
Comment (JamesPHoughton) [inline cypress/e2e/Mobile_Check.js:27]: we only have 1 player in this test, so you can remove line 27 `cy.wrap(playerKeys, { log: false }).each((playerKey) => {` and it's trailing brace, and just have the inside code.

**Review by JamesPHoughton — CHANGES_REQUESTED** [review]
Code looks nice, but the `Normal Paths` and `Batch_Canceled from intro steps` should be working. Did we change something?

**Comment by sky1113** [comment]
Ok, I think this should be good to go, let me know if you find anything else!

**Review by JamesPHoughton — APPROVED** [review]
Thanks @sky1113 - this looks great. =)

**Pull request merged**
